sidebar issues

JsonLitz opened the pull request:
ISSUE #1 Inconsistent Sidebar Styles -

Fixed typo in lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb

Styles were not being applied to the particular elements because the  class was spelled incorrectly. Changing the class name allows the correct styles to be applied to their respective html elements.

jasondotlee committed:
sidebar issues

justincastilla commented on app/assets/stylesheets/user-styles.css:
```diff
@@ -30,6 +30,8 @@ img.right {
 #sidebar .sidebar-title {
   font-family: monospace;
+
+
```
Why is there so much room in here?

jasondotlee committed:
remove trailing whitespace in styles

nathanallen commented:
This looks good and fixes the font of the title, but I'm still seeing inconsistent bullets.

jasondotlee committed:
fixed bullet issue

JsonLitz commented:
The the archived bullets had the class "sidebar_body" instead of "sidebar-body". Thus they were unable to get the correct styles applied to them. Changed the underscore to a hyphen and fixed the issue
